source/includes/Programs/state_conversions.md

```diff
@@ -0,0 +1,362 @@
+
+### Properties
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *[concept](#concepts)* | `Concept_UUID` | Concept resource UUID
+    *[programWorkflow](#program-workflows)* | `ProgramWorkflow_UUID` | ProgramWorkflow resource UUID
+    *[programWorkflowState](#program-workflows)* | `ProgramWorkflowState_UUID` | ProgramWorkflowState resource UUID
+
+
+## Update a Concept State Conversion
+
+> Update a Concept State Conversion
+
+```shell
+POST /stateconversion/:uuid
+{
+    "concept": "116128AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+    "programWorkflow": "54524222-f41d-4d0d-b688-0fd5331377b4",
+    "programWorkflowState": "65b552f7-a768-4986-b00c-28f765625450"
+}
+```
+
+```java
+OkHttpClient client = new OkHttpClient().newBuilder()
+  .build();
+MediaType mediaType = MediaType.parse("application/json");
+RequestBody body = RequestBody.create(mediaType, "{\"concept\": \"116128AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\",\"programWorkflow\": \"54524222-f41d-4d0d-b688-0fd5331377b4\",\"programWorkflowState\": \"65b552f7-a768-4986-b00c-28f765625450\"}");
+Request request = new Request.Builder()
+  .url("/openmrs/ws/rest/v1/stateconversion/:uuid")
+  .method("POST", body)
+  .addHeader("Authorization", "Basic YWRtaW46QWRtaW4xMjM=")
+  .addHeader("Content-Type", "application/json")
+  .addHeader("Cookie", "JSESSIONID=ED9DBD5CFD355A973EFFECD642D8331D")
+  .build();
+Response response = client.newCall(request).execute();
+```
+
+```javascript
+var requestHeaders = new Headers();
+requestHeaders.append("Authorization", "Basic YWRtaW46QWRtaW4xMjM=");
+requestHeaders.append("Content-Type", "application/json");
+requestHeaders.append("Cookie", "JSESSIONID=ED9DBD5CFD355A973EFFECD642D8331D");
+var raw = JSON.stringify({"concept": "116128AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA","programWorkflow": "54524222-f41d-4d0d-b688-0fd5331377b4","programWorkflowState": "65b552f7-a768-4986-b00c-28f765625450"});
+var requestOptions = {
+  method: 'POST',
+  headers: requestHeaders,
+  body: raw,
+  redirect: 'follow'
+};
+fetch("/openmrs/ws/rest/v1/stateconversion/:uuid", requestOptions)
+  .then(response => response.text())
+  .then(result => console.log(result))
+  .catch(error => console.log('error', error));
+```
+
+Update a Concept State Conversion. This method only modifies properties specified in the request. Returns a `404 Not Found` status if conversion does not exist. If not authenticated or authenticated user does not have sufficient privileges, `401 Unauthorized` status is returned.
+
+### Properties
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *[concept](#concepts)* | `Concept_UUID` | Concept resource UUID
+    *[programWorkflow](#program-workflows)* | `ProgramWorkflow_UUID` | ProgramWorkflow resource UUID
+    *[programWorkflowState](#program-workflows)* | `ProgramWorkflowState_UUID` | ProgramWorkflowState resource UUID
+
+
+## Delete a Concept State Conversion
+
+> Delete a Concept State Conversion
+
+```shell
+DELETE /stateconversion/:uuid?purge=true
```

Comment:
Did you forget to rename stateconversion?

Reply:
I did :) Here's a fix: https://github.com/openmrs/openmrs-contrib-rest-api-docs/pull/158/commits/a0fc43c09f22bf697af442af2bb86ac9549c16ea